### ISSUE TITLE
TypeError: Cannot read property 'createResizedImage' of undefined

### ISSUE THREAD

**Issue by hshbaklo** [bamlab/react-native-image-resizer#228]
Still facing this issue on Android device. (no issues on Android emulator).
I tried previous solutions but non worked.
"react": "16.12.0",
"react-native": "0.61.5"
"react-native-image-resizer": "^1.2.0",

**Comment by PierreCapo** [comment]
Hello,
This probably means that strangely your react-native-image-resizer module isn't linked properly (weird that it only happens in your device only but works for emulator). Can you do a:

```js
import { NativeModules }  from "react-native";

console.log(NativeModules
```
And see if you find some react-native-image-resizer there as output ?

**Comment by cnps** [comment]
@PierreCapo I have same issue on Android

"react": "16.8.6",
"react-native": "0.60.6",
"react-native-image-resizer": "^1.2.0",

I've found no keys related to react-native-image-resizer

**Comment by PierreCapo** [comment]
This means this native module is not properly linked and so is not available from the Javascript world. Can you do a `react-native link react-native-image-resizer` ?

**Comment by hshbaklo** [comment]
@PierreCapo hello, react-native-image-resizer key appears in console when I run  app from android emulator but does not appear when running from android device 
Android version on device: 7.0

**Comment by cnps** [comment]
@PierreCapo I've linked the react-native-image-resizer succesfully. I'm still getting error. Seems like it's both on android android and iPhone. However, it works on emulator!

**Comment by cnps** [comment]
@PierreCapo I noticed we used image-resizer 1.2 with RN 0.60, so I switched to 1.1. It works on android, but i'm still getting the createResizedImage of undefined error on ios.

**Comment by hshbaklo** [comment]
issue fixed with release 1.2.1

**Comment by PierreCapo** [comment]
@hshbaklo cool happy to know !
Thanks for the follow-up
